### PR TITLE
[CherryPick] Cookbook "How to create a unit of measure"

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: cookbook/catalog_structure/how_to_add_a_custom_unit_of_measure.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        // ...
+    }
+
+Configure our unit of measure
+-----------------------------
+	
+Create a file ``Resources/config/measure.yml`` in our Bundle to configure the unit of measure:
+
+.. code-block:: yaml
+
```
Comment (willy-ahva) [inline cookbook/catalog_structure/how_to_add_a_custom_unit_of_measure.rst]: extra spaces ?

File: cookbook/catalog_structure/how_to_add_a_custom_unit_of_measure.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+-------------------------
+
+Create the files ``Resources/translations/messages.en.yml`` and ``Resources/translations/jsmessages.en.yml`` in our Bundle to translate our custom unit of measure.
+
+These two files contain this translation:
+
+.. code-block:: yaml
+
+	MICROMETER: micrometer
+
```
Comment (willy-ahva) [inline cookbook/catalog_structure/how_to_add_a_custom_unit_of_measure.rst]: extra spaces

File: cookbook/catalog_structure/how_to_add_a_custom_unit_of_measure.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.. code-block:: yaml
+	
+    measures_config:
+    Length:
+        units:
+            MICROMETER:
+                convert: [{'mul': 0.000001}]
+                symbol: μm
+
+Here, we just add the "MICROMETER" unit with its conversion rules from it to standard unit. To have equivalent to 1 micrometer in meters, you must multiply by 0,000001. A symbol is also required to define the unit format to display.
```
Comment (MarieMinasyan) [inline cookbook/catalog_structure/how_to_add_a_custom_unit_of_measure.rst]: Remove _from it_ and put _to a standard unit_

**Comment by mickaelandrieu** [comment]
@willy-ahva if I update, the commit need to be cherry-picked in master branch :)

**Comment by mickaelandrieu** [comment]
Comments adressed, thak you everyone :)

**Pull request merged**